packages/sanity/src/core/releases/tool/ReleasesTool.tsx

```diff
@@ -0,0 +1,14 @@
+import {useRouter} from 'sanity/router'
+
+import {BundleDetail} from './BundleDetail'
+import BundlesOverview from './BundlesOverview'
+
+export function ReleasesTool() {
+  const router = useRouter()
+
+  const {bundleId} = router.state
```

Comment:
I'm not sure if this is really the best way of managing routing, but it was how Scheduled Publishing seemed to do it and I couldn't find any other examples